lacework-agent/values.yaml

```diff
@@ -50,6 +51,10 @@ laceworkConfig:
   # [Required] An access token is required before running agents.
   # Visit https://<LACEWORK UI URL> for eg: https://lacework.lacework.net
   accessToken:
+  # [Optional] Anonymize incoming IP addresses collected by the agent
+  anonymizeIncoming:
+    # [Optional] Use the following netmask to anonymize addresses
+    netmask:
```

Comment:
@jprakash-lw why are these documented as optional when they were added to the schema as required?